examples/statistics/histogram_demo_cumulative.py

```diff
@@ -1,29 +1,67 @@
 """
-Demo of the histogram (hist) function used to plot a cumulative distribution.
+==========================================================
+Demo of using histograms to plot a cumulative distribution
+==========================================================
+
+This shows how to plot a cumulative, normalized histogram as a
+step function as means of visualization the empirical cumulative
+distribution function (CDF) of a sample. We also use the `mlab`
+module to show the theoretical CDF.
+
+A couple of other options to the `hist` function are demostrated.
+Namely, we use the `normed` parameter to normalize the histogram and
+a couple of different options to the `cumulative` parameter. Normalizing
+a histogram means that the counts within each bin are scaled such that
+the total height of each bin sum up to 1. Since we're showing the
+cumulative histogram, the max value at the end of the series is 1.
```

Comment:
WRT the previous comment, if `normed=True` _and_ `cumulative=True`, then the normalization is not what either of us previously said, but it is such that the max value is 1. That's _different_ behaviour from just `normed=True`, though.